packages/apps/graph-client/src/pages/block/overview/[hash].tsx

```diff
@@ -107,12 +107,12 @@ const Block: React.FC = () => {
                     <strong>Confirmation Depth</strong>
                   </Table.Td>
                   <Table.Td>
-                    {configData?.graphConfiguration?.maximumConfirmationDepth
-                      ? data.block.confirmationDepth ===
-                        configData.graphConfiguration?.maximumConfirmationDepth
-                        ? `> ${data.block.confirmationDepth - 1}`
-                        : data.block.confirmationDepth
-                      : data.block.confirmationDepth}
+                    {!configData?.graphConfiguration
+                      ?.maximumConfirmationDepth ||
+                    data.block.confirmationDepth <
+                      configData.graphConfiguration.maximumConfirmationDepth
+                      ? data.block.confirmationDepth
+                      : `>${data.block.confirmationDepth}`}
```

Comment:
Small and probably unimportant but: if the confirmation depth limit is set to 6 and a block's depth is 6, the UI will display >6 which is technically incorrect.

Reply:
This is true, but if the max confirmation depth is set to 6, then it will never display >6 because it doesn't calculate further. There will never be a 7.